requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
-flake8
-moto
+flake8==3.5.0
+moto==1.3.3
```

Comment:
Do we have a reason to pin `moto` all the way down to patch version?

Reply:
Can confirm that this doesn't work if I unpin the patch version 😢. I think we should figure this out _after_ getting the security fix out

Reply:
Okay sounds good